### DISCUSSION TITLE
Multiple sources in a single track

### DISCUSSION THREAD

**Discussion by jawwad-dev** [bbc/peaks.js#531] [Q&A]
Does Peek.js support implementing multiple audio sources with different color waves in a single track? I have multiple audio tracks, and I want to render all of them on a single track with a different color for each file. Additionally, I need all of them to be controlled from a single source.

**Comment by chrisn** [comment]
It doesn't at the moment, but I'm open to adding support.  How would you want to loading the multiple waveforms - do you have multiple AudioBuffers, for example?
